docs/management/index-patterns.asciidoc

```diff
@@ -23,79 +18,93 @@ image::images/management-index-read-only-badge.png[Example of Index Pattern Mana
+Clicking *Create index pattern* includes an item for creating a rollup 
+index pattern, if a rollup index is detected in the cluster. You create an 
+index pattern for rolled up data the same way you do for any data.  
 
-To reload the index fields list:
+You can match an index pattern to only rolled up data, or mix both rolled 
+up and raw data to visualize all data together. An index pattern can match 
+only one rolled up index, not multiple. There is no restriction on the 
+number of standard indices that an index pattern can match.
+With a rollup index, the time filter field is the same field used for 
+the rolled up date histogram aggregation.
 
-. Go to the *Settings > Indices* tab.
-. Select an index pattern from the Index Patterns list.
-. Click the pattern's *Reload* button.
+See <<visualize-rollup-data, Creating a visualization using rolled up data>> 
+for more detailed information.
 
 [float]
-[[delete-pattern]]
-== Deleting an Index Pattern
-Deleting an index pattern removes the pattern from the list of Saved Objects in Kibana.
-You will not be able to recover field formatters, scripted fields, source filters, and field popularity data
-associated with the index pattern.
+=== Managing your index patterns
+
+Once you’ve created an index pattern, you’re presented a list of all fields 
+and associated data type in the index.  
+
+[role="screenshot"]
+image:management/index-patterns/images/new-index-pattern.png["Index files and data types"]
+
+From this page, you can perform actions on the index pattern using the 
+controls in the upper right. You can:
 
-Deleting an index pattern breaks all visualizations, saved searches, and other saved objects that reference the pattern.
-Deleting a pattern does not remove any indices or data documents from Elasticsearch.
+* *Make it the default index pattern.* {kib} uses a badge to make users 
+aware of which index pattern is the default. The first pattern 
+you create is automatically designated as the default pattern. The default 
+index pattern is loaded when you view the Discover tab.  
 
-To delete an index pattern:
+* *Reload the index fields list.* You can reload the index fields list to 
+pick up any newly-added fields. Doing so also resets Kibana’s popularity counters 
+for the fields. The popularity counters keep track of the fields 
+you’ve used most often within {kib} and are used to sort fields within lists.
 
-. Go to the *Settings > Indices* tab.
-. Select the pattern you want to remove in the Index Patterns list.
-. Click the pattern's *Delete* button.
-. Confirm that you want to remove the index pattern.
+* *Delete the index pattern.* This action removes the pattern from the list of 
+Saved Objects in {kib}. You will not be able to recover field formatters, 
+scripted fields, source filters, and field popularity data associated with the index pattern.
++
+Deleting an index pattern breaks all visualizations, saved searches, and 
```

Comment:
Can you add another bullet about managing fields here and link to https://www.elastic.co/guide/en/kibana/master/managing-fields.html.